aspnet/signalr/overview/performance/scaleout-with-sql-server.md

```diff
@@ -104,15 +104,15 @@ Prepare your Windows Server instances to deploy the SignalR application.
 
 Add the IIS role. Include "Application Development" features, including the WebSocket Protocol.
 
-![Window titled 'Add Roles and Features Wizard'. On the 'Server Roles' option, 'WebSocket Protocol' is selected.](scaleout-with-sql-server/_static/image4.png)
+![Window titled Add Roles and Features Wizard On the Server Roles option WebSocket Protocol is selected.](scaleout-with-sql-server/_static/image4.png)
```

Comment:
How about, "Screenshot that shows Add Roles and Features Wizard.  Web Socket Protocol is selected."